<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Evaluation</class>
 <widget class="QDialog" name="Evaluation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="preFrame">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>720</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>上一帧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextFrame">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>720</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一帧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>720</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改点位置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>770</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>保存该姿势</string>
   </property>
  </widget>
  <widget class="QLabel" name="keypointPic">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>1120</width>
     <height>630</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="scaleSlider">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>60</y>
     <width>22</width>
     <height>630</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>1120</width>
     <height>630</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1118</width>
      <height>628</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
